cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/AbstractQueueConsumer.java

```diff
@@ -306,6 +306,11 @@ private boolean fetchFromCache(SortedMap<byte[], SimpleQueueEntry> entries, int
     return true;
   }
 
+  public static long getWritePointer(byte[] rowKey, int queueRowPrefixLenght) {
```

Comment:
Don't add random static method like this to non-util type class. Move this to `QueueEntryRow` as

`getEnqueueWritePointer(byte[] rowKey, byte[] queueRowPrefix)`.